CHANGELOG/CHANGELOG-4.0.md

```diff
@@ -1,3 +1,31 @@
+# Release notes for v4.0.1
+
+[Documentation](https://kubernetes-csi.github.io)
+
+# Changelog since v4.0.0
+
+## Changes by Kind
+
+### Other (Cleanup or Flake)
+
+- --http-endpoint will now correctly be used for the metrics server address when --metrics-address is not provided. (#497, @ggriffiths)
+- Changed the webhook image from distroless/base to distroless/static. (#550, @WanzenBug)
```

Comment:
This needs some manual work for the PR and git id link.  See the following example:
https://github.com/kubernetes-csi/external-snapshotter/pull/635/files